typescript/cli/examples/submit/strategy/gnosis-ica-strategy.yaml

```diff
@@ -0,0 +1,13 @@
+chain: avalanche
+submitter:
+  type: gnosisSafe
+  props:
+    chain: avalanche
```

Comment:
seems like the `submitter.props.chain` is always equal to the top level chain? can we omit one?

Reply:
i'd rather keep the dupe here if you're alright with it– it's just gnosis where we want to keep a chain set on the submitter (used to validate all txns are compatible with the gnosis safe), and the high-level chain here is the standard for the strategy files